### ISSUE TITLE
Can subnet of kubernetes cluster be used for cosmos db subnet?

### ISSUE THREAD

**Issue by V4A001** [Azure/azure-cli#16842]
Can subnet of kubernetes cluster be used for cosmos db subnet?
Or do I need to create a new second subnet?
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ceafa346-5ac6-9a5b-b36b-fb6a91449ae0
* Version Independent ID: 12dc1540-3a61-aece-0ef0-0d4c192d930e
* Content: [az cosmosdb network-rule](https://docs.microsoft.com/en-us/cli/azure/cosmosdb/network-rule?view=azure-cli-latest)
* Content Source: [latest/docs-ref-autogen/cosmosdb/network-rule.yml](https://github.com/MicrosoftDocs/azure-docs-cli/blob/master/latest/docs-ref-autogen/cosmosdb/network-rule.yml)
* Service: **cosmos-db**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by yonzhan** [comment]
cosmosdb

**Comment by AlecDusheck** [comment]
Are you trying to expose the cosmos db to the aks vnet?

**Comment by V4A001** [comment]
yes, I found this article, https://docs.microsoft.com/en-us/azure/cosmos-db/how-to-configure-vnet-service-endpoint

I am not a network engineer, just want to code :-). I find it difficult to understand and for that reason I am asking. My aks has already a subnet (Azure Network instead of Kubenet)

**Comment by AlecDusheck** [comment]
We've configured our CosmosDB accounts to only be exposed to a private network. I've only tested this with Kubenet though.
<img width="728" alt="Screen Shot 2021-02-14 at 6 30 33 PM" src="https://user-images.githubusercontent.com/22039172/107893851-14341d00-6ef3-11eb-99ae-657522d97b06.png">

**Comment by hemantkr** [comment]
The following doc confirms this can be done:
https://docs.microsoft.com/en-us/azure/aks/concepts-network#azure-virtual-networks
![image](https://user-images.githubusercontent.com/5783237/108435086-9c6d3780-71fd-11eb-9dfd-7c1710afabc1.png)

**Comment by MehaKaushik** [comment]
Tagging AKS team since have more context on this.
